npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/registry_finder.rb

```diff
@@ -203,19 +208,20 @@ def unique_registries(registries)
         end
 
         # rubocop:disable Metrics/PerceivedComplexity
+        # rubocop:disable Metrics/AbcSize
         def global_registry
           return @global_registry if defined? @global_registry
 
           npmrc_file&.content.to_s.scan(NPM_GLOBAL_REGISTRY_REGEX) do
             next if Regexp.last_match[:registry].include?("${")
 
-            return @global_registry = Regexp.last_match[:registry].strip
+            return @global_registry = Regexp.last_match[:registry].strip.gsub(/\s+/, "%20")
```

Comment:
I think this change is unnecessary and it just a leftover from fixing merge conflicts to get the PR up to date?

Reply:
I reason why I added this is, for fixing any spaces in global uri